datamade/just-spaces
# Add import_data script to load ACS data into the database

## Overview

Closes #99.

Load ACS data into the database with an `import_data` script.

## Testing Instructions

* Run `python manage.py import_data` and confirm that it runs successfully
* `psql` into the database and inspect the `acs_` tables to confirm that they look good
